asset/sass/styles.scss

```diff
@@ -0,0 +1,9 @@
+@import url(https://fonts.googleapis.com/css?family=PT+Sans:400,400italic,700,700italic&subset=latin-ext,latin);
+@import url(https://fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,700italic,400,300,600,700&subset=latin-ext,latin);
+@import url(https://fonts.googleapis.com/css?family=Libre+Baskerville:400,400italic,700&subset=latin,latin-ext);
+
+@import 'base';
+@import 'zf-web';
+@import 'zf';
+@import '../node_modules/prismjs/themes/prism-okaidia.css';
```

Comment:
It's fine, CSS are common for all, you can compare here:
https://github.com/PrismJS/prism/blob/gh-pages/themes/prism-okaidia.css
https://github.com/zendframework/zf-mkdoc-theme/blob/master/theme/css/prism-zf.css